qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -16,14 +16,16 @@
 
 import re
 import warnings
+from collections import defaultdict
 from itertools import product
-from typing import Optional, Union, cast
+from typing import Iterator, Optional, Union, cast
```

Comment:
`collections.abc.Iterator` would be better

Reply:
Done.